docs/PLUGIN_CATALOG.md

```diff
@@ -109,6 +113,7 @@ This file is automatically generated. If you would like to add to the plugin lis
 | [RabbitMQ](https://github.com/intelsdi-x/snap-plugin-publisher-rabbitmq) | [Intel](https://github.com/intelsdi-x) | Publishes snap metrics to RabbitMQ | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-publisher-rabbitmq.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-publisher-rabbitmq) | [release](https://github.com/intelsdi-x/snap-plugin-publisher-rabbitmq/releases) |
 | [Riemann](https://github.com/intelsdi-x/snap-plugin-publisher-riemann) | [Intel](https://github.com/intelsdi-x) | Publishes snap metrics to Riemann | [![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-publisher-riemann.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-publisher-riemann) | [release](https://github.com/intelsdi-x/snap-plugin-publisher-riemann/releases) |
 | [SignalFx](https://github.com/opsvision/snap-plugin-publisher-signalfx) | [OpsVision](https://github.com/opsvision) | Publishes metrics to SignalFx. | [![Build Status](https://travis-ci.org/opsvision/snap-plugin-publisher-signalfx.svg?branch=master)](https://travis-ci.org/opsvision/snap-plugin-publisher-signalfx) |  |
+| [Slack](https://github.com/marcin-krolik/snap-plugin-publisher-slack) | [marcin-krolik](https://github.com/marcin-krolik) | No description available. |  |  |
```

Comment:
@marcin-krolik, could you add repo description to https://github.com/marcin-krolik/snap-plugin-publisher-slack?
It is used to generate the content of column "Description" in PLUGIN_CATALOG.md. It might be in the form _"Publishes snap metrics to [...]_. 

@marcin-krolik, once again thank you for contribution. It's great to have a publisher to Slack. I assume it will be very useful for alerting.